java/org/contikios/cooja/Simulation.java

```diff
@@ -915,22 +920,6 @@ public MoteType[] getMoteTypes() {
     return types;
   }
 
-  /**
-   * Returns mote type with given identifier.
-   *
-   * @param identifier
-   *          Mote type identifier
-   * @return Mote type or null if not found
-   */
-  public MoteType getMoteType(String identifier) {
```

Comment:
I think this method might be useful, for example, when adding motes via a simulation script.

Reply:
Dropped the commit.